src/jats/importer/jats-reference-parser.ts

```diff
@@ -125,59 +107,47 @@ export const jatsReferenceParser = {
       const year = getTrimmedTextContent(element, 'year')
 
       if (year) {
-        bibliographyItem.issued = buildBibliographicDate({
+        bibliographyItem.issued = {
           'date-parts': [[year]],
-        })
+          _id: generateID(ObjectTypes.BibliographicDate),
+          objectType: ObjectTypes.BibliographicDate,
+        }
       }
 
       const doi = getTrimmedTextContent(element, 'pub-id[pub-id-type="doi"]')
 
       if (doi) {
-        bibliographyItem.DOI = doi
+        bibliographyItem.doi = doi
       }
 
-      // TODO: handle missing person-group-type?
-      // TODO: handle contrib-group nested inside collab
-      // TODO: handle collab name
-
       const authors: BibliographicName[] = []
 
       authorNodes.forEach((authorNode) => {
-        const name = buildBibliographicName({})
-
+        const name: BibliographicName = {
```

Comment:
Can we continue using `buildBibliographicName`?

Reply:
Sure.